src/fulltrade/FullTradeCard.js

```diff
@@ -57,11 +57,11 @@ export default class BaseTradeCard extends Component {
                     assetName={selectedSymbol}
                     onClosePanel={() => actions.removeTrade(index)}
                 />
-                {tick && <TradeChart
+                {(tick && Object.keys(tick).length > 0) && <TradeChart
                     symbol={selectedSymbol}
                     history={tick}
                 />}
-                <FullTradeParams {...this.props} />
+                {(contract && Object.keys(contract).length > 1) && <FullTradeParams {...this.props} />}
```

Comment:
We are trying to patch a bug by not fixing the bug I suspect.
Two other questions:
* Is contracts === undefined a bug?
* if it isn't possibly FullTradeparams can still handle a null contract by having a defaultValues = { contract: {} };